modules/atomic-widgets/dynamic-tags/dynamic-tags-schemas.php

```diff
@@ -67,6 +68,12 @@ private function convert_control_to_prop_type( array $control ) {
 				->enum( array_keys( $control['options'] ?? [] ) );
 		}
 
+		if ( 'switcher' === $control_type ) {
+			$default = $control['default'];
+			return Boolean_Prop_Type::make()
+				->default( 'yes' === $default || true === $default ? true : false );
```

Comment:
```suggestion
				->default( 'yes' === $default || true === $default );
```
this means the same, unless im missing something